numpy/lib/tests/test_mixins.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 from numpy.testing import assert_, assert_equal, assert_raises
+import pytest
```

Comment:
Not sure this was necessary, but no real harm here.

Reply:
oops, this is indeed totally unnecessary